scripts/extent/shpToBbox.yml

```diff
@@ -0,0 +1,14 @@
+script: shpToBbox.R
+description: "This script convert an extent (shapefile format) to a bbox in the WGS84 format."
+inputs:
```

Comment:
add proj/CRS parameter here

Reply:
I am using ./SDM/extentToBbox.yml to convert spatial object to a bbox. So, I am not sure who is using this shpToBbox function.  May be Guillaume in the SDMs?

Reply:
I am using ./SDM/extentToBbox.yml to convert spatial object to a bbox. So, I am not sure who is using this function.  May be Guillaume?